src/main/scripts/bin.js

```diff
@@ -32,3 +34,27 @@ console.log(
 )
 
 console.log(`cd ${repoName} && npm run dev`)
+
+console.log(`Author: ${authorName}`)
+
+// Dynamically update package.json
+const packageJsonPath = `${repoName}/package.json`
+
+try {
+  const packageJson = JSON.parse(fs.readFileSync(packageJsonPath, 'utf8'))
+
+  // Update package.json fields
+  packageJson.name = repoName
+  packageJson.author = authorName
+  packageJson.version = '0.0.1'
+
+  // Write the updated package.json back to the file
+  fs.writeFileSync(packageJsonPath, JSON.stringify(packageJson, null, 2))
+
+  console.log('Updated package.json with dynamic values.')
+} catch (error) {
+  console.error('Failed to update package.json: ', error)
+}
+
+export { repoName, authorName }
```

Comment:
why are you exporting?

Reply:
Yes that is unecessary line of code , it will work without that also

Reply:
I wiil remove unecessary line of code and push it again

Reply:
Line no. 56 code in using try and catch method to  handle any  error if occur